notebooks/203-meter-reader/203-meter-reader.ipynb

```diff
@@ -288,8 +288,8 @@
     "\n",
     "def roi_process(input_images, target_size, interp=cv2.INTER_LINEAR):\n",
     "    \"\"\"\n",
-    "    Prepare the roi image of detecitonresults data\n",
-    "    Pre-proccesing the input data for segmentation task\n",
+    "    Prepare the roi image of deteciton results data\n",
```

Comment:
"deteciton results" should be "detection results"

Reply:
Sorry my mistake